Authorize/v2/_tucson.yaml

```diff
@@ -1 +1,3 @@
+host: love1.tu.lsst.org
+port: 80
```

Comment:
Please consider putting the port in _init.yaml. Surely it'll be the same at all sites?

I see that value is 0 right now, but 0 is only for unit tests, and I think they should be responsible for making that happen, not the config file. (I don't know how simulation mode works, but if a special port is needed, it can handle that internally to the CSC).

Reply:
Good suggestion. And fortunately it doesn't affect the ts_authorize unit tests. Done.